src/cmake-tools.ts

```diff
@@ -401,16 +444,27 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    */
   get allTargetName() { return this._allTargetName(); }
   private async _allTargetName(): Promise<string> {
-    const drv = await this._cmakeDriver;
-    return drv.allTargetName;
+    const drv = await this.getCMakeDriverInstance();
+    if (drv) {
+      return drv.allTargetName;
+    } else {
+      return "";
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve('');
```

Reply:
Style guide question